pkg/minikube/audit/row_test.go

```diff
@@ -97,7 +97,7 @@ func TestRow(t *testing.T) {
 	t.Run("toFields", func(t *testing.T) {
 		got := r.toFields()
 		gotString := strings.Join(got, ",")
-		want := []string{c, a, p, u, v, stFormatted, etFormatted}
+		want := []string{c, a, p, u, v, stFormatted, id}
```

Comment:
@spowelljr Made changes as suggested 
- This test was failing before because it was not expecting `endTime` to be returned but func `toFields()` was returning it, so I removed `endTime` in the function. Do we need the func `toFields()` to also return `endTime` or is it fine?

Reply:
We need `toFields()` to return `endTime` because that's what we show to the user.